bonnie-rest/src/main/java/com/cgi/hexagon/bonnierest/OrderController.java

```diff
@@ -49,6 +49,17 @@ public ResponseEntity<List<Order>> getAllOrders() {
         }
     }
 
+    @GetMapping(path="/getUnclaimed")
+    public ResponseEntity<List<Order>>  getUnclaimed() {
+        try {
+            List<Order> orders = orderService.getUnclaimed();
+            return ResponseEntity.ok(orders);
+        }catch (Exception e) {
+            System.out.println(e.getMessage());
```

Comment:
we should not use standard output... either add logging or just do not print anything

Reply:
I don't think I understand.. Is this just about the try-catch print message or the whole output? We do want to return all the orders based on the status right?

Reply:
Yes I am just objecting to the System.out.println